packages/components/gulpfile.js

```diff
@@ -338,6 +341,7 @@ const buildStyles = prod => {
       .pipe(
         sass({
           outputStyle: prod ? 'compressed' : 'expanded',
+          fiber: Fiber,
```

Comment:
IIUC this is for `dart-sass`, is it correct?

Reply:
Yep ... without `fiber` the compile time is significantly slower when using the async compiler.